torch/_dynamo/symbolic_convert.py

```diff
@@ -775,8 +775,11 @@ def wrapper(self: "InstructionTranslatorBase", inst: Instruction):
                     unimplemented_v2(
                         gb_type="Graph break under GenericContextWrappingVariable",
                         context="",
-                        explanation="Attempted to graph break in an active context manager that doesn't support graph breaking.",
-                        hints=[],
+                        explanation="Attempted to graph break in an active context manager(s) that doesn't support graph breaking.",
+                        hints=[
+                            "Move the offending context manager(s) to outside the compiled region.",
```

Comment:
An alternative is to get rid of the graph break inside the context manager. This might be easier to try then moving the context manager